src/plugins/orangehrmAttendancePlugin/Dao/AttendanceDao.php

```diff
@@ -363,20 +391,23 @@ private function getAttendanceReportQueryBuilderWrapper(
             // from date is not null and to date is null
             $q->leftJoin('employee.attendanceRecords', 'attendanceRecord', Expr\Join::WITH, $q->expr()->andX(
                 $q->expr()->gte('attendanceRecord.punchInUserTime', ':fromDate')
-            ));
+            )
+            );
```

Comment:
I am not sure what is convention here in PHP, but it looks janky for me to be like this, could we either revert this change or change the call to the topmost function to be multiline

Reply:
this is my subconsciousness spamming alt + f 😄 . will revert